src/main/asciidoc/java/index.adoc

```diff
@@ -362,9 +362,10 @@ with respect to concurrency, scaling and deployment.
 
 To use this model, you write your code as set of *verticles*.
 
-Verticles are chunks of code that get deployed and
-run by Vert.x. Verticles can be written in any of the languages that Vert.x supports and a single application
-can include verticles written in multiple languages.
+Verticles are chunks of code that get deployed and run by Vert.x. Vert.x will create N event loops threads
+(where N by default is core*2) by default. When you deploy a verticle,
+it will be deployed on one of theses event loops. Verticles can be written in any of the languages that Vert.x supports
```

Comment:
The verticle is not deployed _on_ an event loop. It is _associated_ to an event loop when the verticle is created.

Also, this is only true for standard (i.e.non worker) verticles.